llama_stack/cli/download.py

```diff
@@ -343,7 +343,7 @@ def _hf_download(
             "You can find your token by visiting https://huggingface.co/settings/tokens"
         )
     except RepositoryNotFoundError:
-        parser.error(f"Repository '{repo_id}' not found on the Hugging Face Hub.")
+        parser.error(f"Repository '{repo_id}' not found on the Hugging Face Hub or incorrect Hugging Face token.")
```

Comment:
Can't `incorrect Hugging Face token` be caught separately?

Reply:
HfHubHTTPError << RepositoryNotFoundError(401 Client Error)  << GatedRepoError(403 Client Error)
`RepositoryNotFoundError` inherit from `HfHubHTTPError`, it should cover to detect the `incorrect token`, but just say `not found`, seems cannot describe it better.(pls correct if i am wrong.)

```
https://github.com/huggingface/huggingface_hub/blob/290aa26f22d415853fd9e3560b9cc616821d2f73/src/huggingface_hub/errors.py#L174
```

Reply:
Thanks for the explanation.